tasks/db_deploy/test/manual_tests/sql/orca_schema_v3/remove.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE orca.recovery_file
+	DROP COLUMN multipart_chunksize_mb
```

Comment:
Do we also need to revert the schema_versions table to only have a version 2 entry?

Reply:
Ah, that was just created in the previous migration. Fixed.